test/data/case5_pst.json

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
In the interest of keeping maintenance simple, the PowerModels test data are all kept in the Matpower format.  You can use the generic [Matpower extensions feature](https://lanl-ansi.github.io/PowerModels.jl/stable/network-data.html) to get the additional data you need into the PowerModels data dictionary.